Example/GiniVision/SelectAPIViewController.swift

```diff
@@ -178,75 +166,58 @@ class SelectAPIViewController: UIViewController {
         })
     }
     
-    func show(_ result: GINIResult, fromDocument document: GINIDocument) {
-        if let _ = analysisDelegate {
-            analysisDelegate = nil
-            present(result, fromDocument: document)
-        }
-    }
-    
     func present(_ result: GINIResult, fromDocument document: GINIDocument) {
         let payFive = ["paymentReference", "iban", "bic", "paymentReference", "amountToPay"]
         let hasPayFive = result.filter { payFive.contains($0.0) }.count > 0
         
         let storyboard = UIStoryboard(name: "Main", bundle: nil)
         if hasPayFive {
```

Comment:
I'd separate the two cases in different methods, this one is getting really big.

Also, the `hasPayFive` name makes me thing all of them need to be there.

Also, Pay Five is something we use internally, but I don't expect others to know what it means. Why not just `hasExtractions`?

Reply:
It is getting really big yes 👍